### DISCUSSION TITLE
I dont get my benefits after i got approved.

### DISCUSSION THREAD

**Discussion by Chabick** [community/community#62750] [GitHub Education] [answered]
### Hi everyone,

I got approved 2 days ago but never got the benefits. It says "Congrats! Your request for [@DAOCMINECRAFT](https://github.com/DAOCMINECRAFT) was approved and your benefits will be available soon." Please help

**Comment by prabhjotschugh** [comment, marked as answer]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!

**Reply by Chabick** [reply]
Thanks

**Reply by goelarnav12** [reply]
Its been more than 2 months after getting approved , ut I still cant seem to get any benefits, any idea what i can do?

**Comment by Spixmacaw** [comment]
Hi guys,
I got approved 2 days ago but never got the benefits. It says "Congrats! Your request for (https://github.com/DAOCMINECRAFT) was approved and your benefits will be available soon." Please help

A suggestion...(optional)
No response

**Reply by prabhjotschugh** [reply]
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.

**Comment by amitvtcbb** [comment]
As a teacher, My request was approved from 30 May,2023. Still not got benefit. when I am going for reverification I show below message regarding verification and continue button is disable.

"Hi,! You were last verified as a teacher on May 30, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

What can I do in such situation?

**Comment by smasongarrison** [comment]
I also have this problem, and have not received a reply from support even though I raised this question in september

**Comment by ibgarrido** [comment]
I have the same problem. I was verified at October, 1 but i never got the benefits neither the email confirmation.
